config/ut-translator.yaml

```diff
@@ -1,47 +1,51 @@
 ---
-api: 'https://translation.usertesting.com/graphql'
```

Comment:
Sure this is not needed?

Reply:
I tested it, it's not needed and @fcano-ut told me is not needed anymore

Reply:
Indeed not needed (reference: https://user-testing.atlassian.net/l/cp/nSdh0Kcq)